apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java

```diff
@@ -179,6 +184,13 @@ public void onChannelBoxClicked(View view) {
         mAudioOutTester.setChannelEnabled(channelIndex, checkBox.isChecked());
     }
 
+    @Override
+    protected void setStreamControlByAttributes(int usage, int contentType) {
+        if (mShouldSetStreamControlByAttributes.isChecked()) {
+            super.setStreamControlByAttributes(usage, contentType);
```

Comment:
A set() method should set().
We should condition the call to the set() method.
The method that is conditional might be called "maybeSet.....".

Reply:
Thanks! Done!